multidim-interop/impl/java/v0.0.1/Makefile

```diff
@@ -1,5 +1,5 @@
 image_name := java-v0.0.1
-commitSha := 6fad470c92a5c4ab368bc615e64dbf40b5ff9ccb
+commitSha := 967b8f2adcaaaf88204939c1f9eeeadf062ce364
```

Comment:
I am unable to find this commit on https://github.com/Peergos/nabu/commits/master. Am I missing something? Mind linking to it directly?

Reply:
Ah looks like it's on a branch [fix/test-plans-latest](https://github.com/Peergos/nabu/compare/fix/test-plans-latest) https://github.com/Peergos/nabu/commit/967b8f2adcaaaf88204939c1f9eeeadf062ce364

Reply:
Friendly ping @kevodwyer. How about we wait for the above to make it into `master` before merging here?